lang/cs/Org.Apache.REEF.Network/NetworkService/WritableNetworkService.cs

```diff
@@ -126,7 +126,7 @@ public void Register(IIdentifier id)
             NamingClient.Register(id.ToString(), _remoteManager.LocalEndpoint);
 
             // Create and register incoming message handler
-            var anyEndpoint = new IPEndPoint(_remoteManager.LocalEndpoint.Address, 0);
+            var anyEndpoint = new IPEndPoint(IPAddress.Any, 0);
```

Comment:
Shouldn't this also use the `TcpPortProvider`? Shall we do that in another JIRA / PR?